site/docs/bazel-and-java.md

```diff
@@ -47,17 +51,34 @@ The resulting binaries are compatible with locally installed JVM in system
 libraries, which means the resulting binaries depend on what is installed on the
 machine.
 
+To configure the JVM used for execution and testing use `--java_runtime_version`
+flag. It's default value is `local_jdk`.
```

Comment:
Remove "It's" and just use "The default value is ...".

Reply:
Done.